frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -343,26 +343,26 @@ export const SETTINGS_MAP: SettingSection[] = [
                 component: <ReplayGeneral />,
             },
             {
-                id: 'replay-network',
-                title: 'Network capture',
-                component: <NetworkCaptureSettings />,
+                id: 'replay-triggers',
+                title: 'Recording conditions',
```

Comment:
i don't think we want to A/B test this but I think worth pinging people from the unsubscribe survey and asking if this would have helped maybe

i guess we can just look at metrics of how many people are opening this settings page 🤔 

just worried that since we know this has strong correlation with unsubscribe intent if we make this less clear by accident then we'll make someone more likely to unsubscribe when they don't need to

Reply:
I think the "settings wizard" / "get max to help" idea is way more important than twiddling the text though!